ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -286,31 +637,24 @@ def _connection_cache(self) -> Dict[_ConnectionKey, Connection]:
                     session_info=session_info,
                     multiplexer_resource_name=channel_mapping.multiplexer_resource_name,
                     multiplexer_route=channel_mapping.multiplexer_route,
-                    multiplexer_session_info=self._get_multiplexer_session_info_for_resource_name(
-                        channel_mapping.multiplexer_resource_name
+                    multiplexer_session_info=(
+                        self._multiplexer_session_handler._get_multiplexer_session_info_for_resource_name(
+                            channel_mapping.multiplexer_resource_name
+                        )
                     ),
                 )
                 assert key not in cache
                 cache[key] = value
         return cache
 
-    @cached_property
-    def _multiplexer_type_ids(self) -> AbstractSet[str]:
-        return _to_ordered_set(
-            sorted(info.multiplexer_type_id for info in self._multiplexer_session_info)
-        )
+    @property
+    def _multiplexer_session_cache(self) -> Dict[str, object]:
+        return self._multiplexer_session_handler._multiplexer_session_cache
 
     @property
-    @requires_feature(MULTIPLEXER_SUPPORT_2024Q2)
     def multiplexer_session_info(self) -> Sequence[MultiplexerSessionInformation]:
```

Comment:
Keep the feature toggles

Reply:
The feature toggle decorator also serves as documentation. Please keep it on all of the new methods.

Reply:
Got it. I'll have them back.